ghmirror/core/mirror_requests.py

```diff
@@ -32,6 +33,21 @@
 LOG = logging.getLogger(__name__)
 
 
+def _params_in_url_params(url):
+    """
+    Get query parameter from the URL of the request
+    """
+    params = urllib.parse.parse_qs(urllib.parse.urlparse(url).query)
```

Comment:
we already parse the url params at the flask app level, maybe we should just pass them through as a separate argument to the `conditional_requests`:

https://github.com/app-sre/github-mirror/blob/master/ghmirror/app/__init__.py#L87-L91